OpenRA.Mods.Common/Traits/Repairable.cs

```diff
@@ -123,41 +123,19 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 				if (!order.Queued)
 					self.CancelActivity();
 
-				self.SetTargetLine(order.Target, Color.Green);
-				var activities = ActivityUtils.SequenceActivities(self,
-					movement.MoveToTarget(self, order.Target, targetLineColor: Color.Green),
-					new CallFunc(() => AfterReachActivities(self, order, movement)));
-
-				self.QueueActivity(new WaitForTransport(self, activities));
-				TryCallTransport(self, order.Target, new CallFunc(() => AfterReachActivities(self, order, movement)));
+				self.QueueActivity(new MoveToResupplier(self, order.Target));
```

Comment:
We really want to be queuing `Resupply` here, and that checks whether it is in the correct position and moves if needed. See https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Activities/HarvestResource.cs#L62 for a recent example